src/compiler/program.ts

```diff
@@ -2631,7 +2631,9 @@ namespace ts {
                     referencedFileName,
                     kind: refFile.kind,
                     index: refFile.index,
-                    file: refFile.file.path

```

Comment:
You shouldnt store these.. These can be calculated and storing them messes with incremental program changes when file graph doesnt change but position of import changes...  Look for createFileDiagnosticAtReference

Reply:
Done, thanks